src/utils/OnboardService.js

```diff
@@ -0,0 +1,33 @@
+import Onboard from 'bnc-onboard'
+
+const networkId = 4;
+const apiUrl = process.env.REACT_APP_API_URL;
+const dappId = '8c078d77-1fe5-4a81-8956-bf62aaadd667';
```

Comment:
Need to replace with another one later.
https://blocknative.com

Reply:
Will we be able to get the network id from their wallet in the future?

Reply:
Yeah, that's possible I guess.
`web3.eth.net.getId()`
Also here are network enums
```
Object.freeze({
  1: 'Ethereum Main',
  3: 'Ropsten',
  4: 'Rinkeby',
  5: 'Goerli',
  25: 'Cronos',
  42: 'Kovan',
  56: 'BSC',
  100: 'XDai',
  137: 'Polygon',
  250: 'Fantom',
  localhost: 'localhost'
});
```

Right now we can only test on Polygon(`137`)?

Reply:
I’ve got a new version I’ll deploy to rinkeby for us to use

Reply:
Thank you!